src/Facade/Services/SernosTransactionService.cs

```diff
@@ -17,13 +18,40 @@ public SernosTransactionService(IRepository<SernosTrans, string> repository, ITr
 
         protected override SernosTrans CreateFromResource(SernosTransactionResource resource)
         {
-            throw new NotImplementedException();
+            return new SernosTrans
+                       {
+                           TransCode = resource.TransCode,
+                           ManualPost = resource.ManualPost,
+                           TransDescription = resource.TransDescription,
+                           Comments = resource.Comments,
+                           SernosTransCounts = resource.SernosTransCounts.Select(
+                               item => new SernosTransCount
+                                           {
+                                               TransCode = item.TransCode,
+                                               SernosCount = item.SernosCount,
+                                               CheckError = item.CheckError,
+                                               CorrectValue = item.CorrectValue,
+                                               CountIncrement = item.CountIncrement,
+                                               CheckErrorMess = item.CheckErrorMess
+                                           }).ToList()
+                       };
         }
 
         protected override void UpdateFromResource(SernosTrans sernosConfig, SernosTransactionResource updateResource)
         {
+            sernosConfig.Comments = updateResource.Comments;
             sernosConfig.TransDescription = updateResource.TransDescription;
             sernosConfig.ManualPost = updateResource.ManualPost;
+            sernosConfig.SernosTransCounts = updateResource.SernosTransCounts.Select(
+                item => new SernosTransCount
+                            {
+                                TransCode = item.TransCode,
```

Comment:
I take it that newing these up means that ef will delete all the lines and create new ones? 
I suppose that's fine on these because sernosCount is the key. Would cause problems if Id was an int and it changed every time.